.github/workflows/tests.yaml

```
@@ -46,3 +47,16 @@ jobs:
 
       - name: Check resources
         run: kubectl get all -n checkpoint-restore-operator-system
+
+      - name: Prepare Checkpoints Directory in Kind Node
+        run: |
+          mkdir -p /var/lib/kubelet/checkpoints
+          chmod 777 /var/lib/kubelet/checkpoints
```

```suggestion
          sudo mkdir -m 777 -p /var/lib/kubelet/checkpoints
```
Since `/var` is owned by root, you would need superuser privileges

---

yap, I am working on it, but getting some problems. idk why operator can't find any checkpoints in that directory while testing locally, in this case I am creating the directory manually but in case where kubelet created it by default the operator works flawlessly.